roles/mirror_registry/molecule/default/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  roles:
+    - role: "mirror_registry"
```

Comment:
[Suggestion] Is this enough to test the role?  Does `mirror-registry install` do some verification? 

If not you could push and pull a tiny container to the registry to verify.

Reply:
Added a job to login and verify that it works. Stopped short of pushing a container since I don't want to build a new one and push it adding another dependency on something on the internet. Maybe there's a consistently available name for the containers quay itself is using and we could push those. But maybe better for a future enhancement.